core/src/main/java/com/novoda/noplayer/AdvertBreak.java

```diff
@@ -0,0 +1,54 @@
+package com.novoda.noplayer;
+
+import java.util.List;
+
+public class AdvertBreak {
```

Comment:
This is our simplified model that acts as a public API for providing the ads to Noplayer.